microservices/biobuddy/biobuddy/services.py

```diff
@@ -5,8 +5,11 @@
 from langchain_core.messages import HumanMessage, SystemMessage, AIMessage
 from langchain_openai import ChatOpenAI
 
+from biobuddy.gpt_researcher import get_report
+import asyncio
+
 chat_model = ChatOpenAI()
-chat_model.model_name = "gpt-4"
+chat_model.model_name = "gpt-4-turbo"
```

Comment:
I'd rather move it to the settings.ini or some sort of this, but let's leave it as it is, me and Timur are going to refactor entire solution anyway